docs/led.md

```diff
@@ -0,0 +1,54 @@
+# LED (Mono color backlight)
+Want your keyboard to shine? Add some lights!
+ 
+## Usage
+At minimum you will need to make sure that this set in either your keymap is importing an MCU directly, or it should be included in the predefined boards if they support them.
```

Comment:
This is probably the most technically intimidating line in beginner-oriented documentation we have.

User keyboard definitions (i.e. the stuff that lives under `user_keymaps`, a bit of a misnomer these days) just extend boards which just extend MCUs. Setting `led_pin` manually is always safe - we can leave a note that some (perhaps keebio boards converted with the kit) boards will have a default set.

Reply:
They will, along with any other stored boards that have known pins, which is why I have included this. More commits incoming to reflect.

Reply:
I really should have put this at the bottom. Not following my own guidelines >_> Solved.